test/python/visualization/references/pass_manager_style.dot

```diff
@@ -1,15 +1,15 @@
 digraph G {
 subgraph cluster_0 {
 fontname=helvetica;
-label="[0] FlowLinear";
+label="[0] linear";
 1 [color=cyan, fontname=helvetica, label=SetLayout, shape=rectangle];
 2 [color=black, fontname=helvetica, fontsize=10, label=layout, shape=ellipse, style=solid];
 2 -> 1;
 }
 
 subgraph cluster_3 {
 fontname=helvetica;
-label="[1] Conditional";
+label="[1] condition";
```

Comment:
```suggestion
label="[1] conditional";
```

Reply:
nit picky but `linear` goes better with `conditional`

Reply:
I could remove `linear` since is the default. The reason for `condition` is because that's how's the pass set declared.

```python
pm.append([pass1, pass2], condition=....)
```

Reply:
after 81149a0e, it looks like this:
<img width="210" alt="Screen Shot 2019-10-23 at 10 07 41 AM" src="https://user-images.githubusercontent.com/766693/67401500-3d51d100-f57d-11e9-940d-b43bdd1d990f.png">